content/docs/howto/php.md

```diff
@@ -27,147 +27,410 @@ for how to run the app.
+`httpd` and `nginx`.
+
+Additional configuration can also be provided via environment variables.
+
+### Provide your own web server configuration file
+If either `httpd` or `nginx` are the selected web server via the
+`BP_PHP_SERVER` environment variable, users can provide their own
+configurations in the form of a server-specific configuration file.
+
+#### Provide `httpd` specific configuration
+In the `httpd` case, user-included configuration can be placed be found in the
+application source directory under `<app-directory>/.httpd.conf.d/*.conf`. This
+is helpful in the event that you want to set custom settings that are not
+configurable via environment variables in the PHP HTTPD buildpack.
+
+#### Provide `nginx` specific configuration
+In the `nginx` case, user-included configuration can be placed in the
+application source directory under `<app-directory>/.nginx.conf.d/`.
+Server-specific configuration should be inside a file named  `*-server.conf`,
+and HTTP configuration should be inside a file with the naming structure
+`*-http.conf`. This is helpful in the event that you want to set custom
+settings that are not configurable via environment variables in the PHP Nginx
+buildpack. Check out [Nginx documentation][external/nginx-conf-docs] for what
+settings can be applied to server and HTTP blocks.
+
+
+### Configure FPM settings
+The PHP buildpack includes support for the [PHP FastCGI Process
+Manager][external/fpm] (FPM) when used in conjunction with a web server. In
+this case, the [PHP FPM buildpack][bp/fpm] will generate FPM configuration for
+you to work with the web server of choice. On top of setting up FPM
+configuration, the buildpack will consider configuration from user provided
+sources (see `Override Default FPM Configuration` section below)
+
+Check out the [PHP Reference documentation]({{< ref
+"docs/reference/php-reference" >}}) for an enumeration of the defaults set for
+FPM.
+
+#### Override Default FPM Configuration
+Users can provide FPM configuration by providing a configuration file in the
+application source directory under `.php.fpm.d/*.conf`.
+
+User-provided configuration will be considered the highest precedence source of
+configuration, and should be provided in an `.ini` compliant format in order to
+be considered by PHP FPM.
+
+### Configure the web directory
+The top-level directory for the web servers to find files to serve can be set
+via the web directory. For the `httpd` and `nginx` server cases can be
+overwritten from the default of `htdocs` via the `BP_PHP_WEB_DIR` environment
+variable at build-time.
+
+For the PHP built-in server case, the default web
+directory is the `/workspace`, but this can also be overrwritten with the
+`BP_PHP_WEB_DIR` environment variable. The PHP built-in server will only pass
+detection if a `*.php` file is found inside of the web directory.
+
+In all cases, the web directory path should be provided relative to the application
+directory root.
```

Comment:
```suggestion
The top-level directory where a web server finds files to serve is
the _web directory_.  In the PHP buildpack, when the web server is 
HTTPD or NGINX, the web directory defaults to `htdocs.` When the web server is
the PHP built-in server, the web directory defaults to `/workspace`.

In all cases, the default web directory can be overridden by setting the
`BP_PHP_WEB_DIR` environment variable at build-time. The value
provided should be a path relative to the application root.

Note: The PHP built-in server will only pass
detection if a `*.php` file is found inside of the web directory.
```

Reply:
Just as an FYI -- overwrite != override. I think the latter is usually the one we want when talking about configuration.

Reply:
This always confuses me. Thank you for pointing it out